.github/renovate.json

```diff
@@ -3,5 +3,11 @@
   "extends": [
     "github>cucumber/renovate-config",
     ":automergeMajor"
+  ],
+  "packageRules": [
+    {
+      "matchPackageNames": ["vscode"],
+      "automerge": false
```

Comment:
Additioanlly, to avoid getting PR's for the calender versions it would be possible to use an `allowedversions` regex.

https://docs.renovatebot.com/configuration-options/#allowedversions

Reply:
Ah. I can't seem to find the Calendar version. I have no idea why Renovate did that.